src/main/java/com/stripe/model/TransferReversalCollection.java

```diff
@@ -12,41 +12,92 @@
 import java.util.Map;
 
 public class TransferReversalCollection extends StripeCollection<Reversal> {
-  public TransferReversalCollection list(Map<String, Object> params)
+  // <editor-fold desc="all">
+  /**
+   * List all reversals.
+   *
+   * @deprecated Use the {@link #list(Map)} method instead.
+   *     This method will be removed in the next major version.
+   */
+  @Deprecated
+  public TransferReversalCollection all(Map<String, Object> params)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return list(params, null);
   }
 
-  public TransferReversalCollection list(Map<String, Object> params,
-                       RequestOptions options) throws AuthenticationException,
+  /**
+   * List all reversals.
+   *
+   * @deprecated Use the {@link #list(Map, RequestOptions)} method instead.
+   *     This method will be removed in the next major version.
+   */
+  @Deprecated
+  public TransferReversalCollection all(Map<String, Object> params,
+                      RequestOptions options) throws AuthenticationException,
       InvalidRequestException, APIConnectionException, CardException,
       APIException {
-    String url = String.format("%s%s", Stripe.getApiBase(), this.getURL());
-    return APIResource.requestCollection(url, params, TransferReversalCollection.class, options);
+    return list(params, options);
   }
+  // </editor-fold>
 
-  @Deprecated
-  public TransferReversalCollection all(Map<String, Object> params)
+  // <editor-fold desc="create">
+  /**
+   * Create a reversal.
+   */
+  public Reversal create(Map<String, Object> params)
+      throws AuthenticationException, InvalidRequestException,
+      APIConnectionException, CardException, APIException {
+    return create(params, null);
+  }
+
+  /**
+   * Create a reversal.
+   */
+  public Reversal create(Map<String, Object> params,
```

Comment:
random again but should we move to a world where you pass the transfer id as a param instead of going through the Collection?

Reply:
Maybe? That would be a much larger change to how nested resources are handled, and I'm not sure it's worth the hassle. I'll open a proof of concept PR so we can discuss the pros and cons.